pex/scie/__init__.py

```diff
@@ -72,6 +74,35 @@ def register_options(parser):
             "https://science.scie.app.".format(lazy=ScieStyle.LAZY, eager=ScieStyle.EAGER)
         ),
     )
+    parser.add_argument(
+        "--scie-busybox",
+        dest="scie_busybox",
+        type=str,
+        default=[],
+        action="append",
+        help=(
```

Comment:
Minor feedback: these docs look good, but they're a bit of a wall of text that'll usually be read in the context of a terminal without any formatting aids, and thus I'd expect most people (even well-intentioned diligent ones) to fail to synthesise the information out of it.

Is it possible to make it easier for users by breaking it up into paragraphs and/or lists?

Reply:
It is not - Argparse doesn't support formatting I know of. I've both avoided spending time on any bespoke argument system to remedy this or turning towards documentation, which need major work going on 10 years now. One or both are coming, but I'm prioritizing robust users so far in favor of righting the Pex ship going on 8 years. I do finally feel close though.

Reply:
Ah I see. Thanks for the background. 👍